package/MDAnalysis/analysis/gnm.py

```diff
@@ -217,17 +219,22 @@ class GNMAnalysis(object):
           `Bonus_groups` is contained in `selection` as this could lead to
           double counting. No checks are applied. Default is ``None``.
 
+    Attributes
+    ----------
+    results : list
+        GNM results per frame:
+            results = [(time,eigenvalues[1],eigenvectors[1]),
+                       (time,eigenvalues[1],eigenvectors[1]), ...]
+
     See Also
     --------
     :class:`closeContactGNMAnalysis`
 
-
     .. versionchanged:: 0.16.0
        Made :meth:`generate_output` a private method :meth:`_generate_output`.
 
     .. versionchanged:: 1.0.0
```

Comment:
Could you also add suitable versionchanged entries?
The impact on user is low but it might interest developers.

Reply:
My local docs look a bit weird. Lets wait how the online version look like...

<img width="728" alt="Screenshot 2021-04-28 at 08 57 18" src="https://user-images.githubusercontent.com/15192133/116360184-f3941780-a7ff-11eb-82c4-b0fb3f52860d.png">